cmd/drone-downstream/config.go

```diff
@@ -64,12 +45,6 @@ func settingsFlags(settings *plugin.Settings) []cli.Flag {
 			EnvVars:     []string{"PLUGIN_PARAMS_FROM_ENV"},
 			Destination: &settings.ParamsEnv,
 		},
-		&cli.StringFlag{
-			Name:        "deploy",
-			Usage:       "Environment to trigger deploy for the respective build",
-			EnvVars:     []string{"PLUGIN_DEPLOY"},
-			Destination: &settings.Deploy,
-		},
```

Comment:
I know you don't have an implementation of this but its going to need to be there in the final version of the plugin.